olive/engine/packaging/packaging_generator.py

```diff
@@ -170,10 +172,15 @@ def _upload_to_azureml_data(
     )
 
 
-def _get_model_info(node: "FootprintNode", model_rank: int):
+def _get_model_info(
+    node: "FootprintNode", model_rank: int, tempdir: Path, model_path: Path, packaging_type: PackagingType
+):
+    model_config = node.model_config
+    if packaging_type == PackagingType.Zipfile:
+        model_config["config"]["model_path"] = str(model_path.relative_to(tempdir))
```

Comment:
>model_path.relative_to(tempdir) [](http://example.com/codeflow?start=51&length=31)

it seems the logic could be moved into the caller. 

From this function logic, the relative to another folder seems be weird since the model_path has assumption starting with the tempdir.

It would be better if we directly pass the relative model_path as parameter to this function and move the relative path extraction logic into _package_candidate_models